provider/datagen/src/provider.rs

```diff
@@ -306,6 +309,28 @@ pub struct SourceData {
     pub(crate) icuexport_dictionary_fallback: Option<Arc<SerdeCache>>,
     #[cfg(feature = "legacy_api")]
     pub(crate) collations: Vec<String>,
+    pub(crate) supported_locales_cache_vec:
+        FrozenMapThrowawayClone<DataKey, Box<Result<Vec<DataLocale>, DataError>>>,
+}
+
+pub(crate) struct FrozenMapThrowawayClone<K, V>(pub(crate) FrozenMap<K, V>);
```

Comment:
I believe `elsa` master has these impls now

Reply:
It does for `sync` at least.

Reply:
`elsa 1.10` should work.

Reply:
Upgraded to elsa 1.10